specs/IFrameMemoryUsage

```diff
@@ -0,0 +1,251 @@
+This event fires when a new frame is created.
+
+
+# Examples
+C++
+```c++
+void ScenarioWebViewEventMonitor::InitializeEventView(ICoreWebView2* webviewEventView)
+{
+    m_webviewEventSource4->add_FrameCreated(
+            Callback<ICoreWebView2FrameCreatedEventHandler>(
+                [this](ICoreWebView2* sender, ICoreWebView2FrameCreatedEventArgs* args)
+                    -> HRESULT {
+                    // ...
+                    // The new frame created event can be recursively initialized
+                    InitializeFrameEventView(webviewFrame, first_level_iframe_depth + 1);
+                    // ...
+                    return S_OK;
+                })
+                .Get(),
+            &m_frameCreatedToken);
+}
+
+void ScenarioWebViewEventMonitor::InitializeFrameEventView(
+    wil::com_ptr<ICoreWebView2Frame> webviewFrame, int depth) 
+{
+    auto frame5 = webviewFrame.try_query<ICoreWebView2Frame5>();
+    if (frame5)
+    {
+        //! [ICoreWebView2Frame:add_ProcessChanged]
+        frame5->add_FrameCreated(
+            Callback<ICoreWebView2NestedFrameCreatedEventHandler>(
+                [this, depth](
+                    ICoreWebView2Frame* sender,
+                    ICoreWebView2FrameCreatedEventArgs* args) -> HRESULT
+                {
+                    std::wstring message;
+                    wil::com_ptr<ICoreWebView2Frame> webviewFrame;
+                    CHECK_FAILURE(args->get_Frame(&webviewFrame));
+                    wil::unique_cotaskmem_string name;
+                    CHECK_FAILURE(webviewFrame->get_Name(&name));
+                    message += L"\"frame\": " + name.get());
+                    InitializeFrameEventView(webviewFrame, depth + 1);
+                    message += L",\"depth\": " + std::to_wstring(depth);
+                    auto webviewFrame5 = webviewFrame.try_query<ICoreWebView2Frame5>();
+                    if (webviewFrame5)
+                    {
+                        //! [ICoreWebView2Frame:get_ProcessId]
+                        unsigned int processId;
+                        CHECK_FAILURE(webviewFrame5->get_ProcessId(&processId));
+                        message +=
+                            L",\"render process id\": " + std::to_wstring((int)processId);
+                    }
+                    return S_OK;
+                })
+                .Get(),
+            &m_frameCreatedToken);
+        //! [ICoreWebView2Frame:add_ProcessChanged]
+        frame5->add_ProcessIdChanged(
+            Callback<ICoreWebView2FrameRenderProcessChangedEventHandler>(
+                [this](
+                    ICoreWebView2Frame* sender,
+                    IUnknown* args) -> HRESULT 
+                {
+                    // Do something when render process id is changed
+                })
+                .Get(),
+            &m_ProcessIdChangedToken);
+    }
+```
+C#
+```c#
+void WebView_HandleIFrames(object sender, CoreWebView2FrameCreatedEventArgs args)

```

Comment:
The C# and C++ sample code should be the same. Currently the C++ code creates a message whereas the C# code creates a frames list.

Reply:
Will refine that later.

Reply:
Refined. I added some comments for both when renderer process id changed/obtained.